tests/test.ts

```diff
@@ -638,7 +611,7 @@ describe.skip('Test clasp run function', () => {
     const result = spawnSync(
       CLASP, ['run', 'myFunction'], { encoding: 'utf8' },
     );
-    expect(result.stdout).to.contain('What is your GCP projectId?');
+    expect(result.stdout).to.contain(`${LOG.ASK_PROJECT_ID}`);
```

Comment:
```ts
expect(result.stdout).to.contain(LOG.ASK_PROJECT_ID);
```

Reply:
Will fix this then merge once build passes.